docs/topics/mta-rn-new-features-7-0-0.adoc

```diff
@@ -0,0 +1,39 @@
+// Module included in the following assemblies:
+//
+// * docs/release_notes/master.adoc
+
+:_content-type: CONCEPT
+[id="rn-new-features-7-0-0_{context}"]
+= New features
+
+
+This section describes the new features of the Migration Toolkit for Applications (MTA) 7.0.0.
+
+.Red Hat Single Sign On (RHSSO) Configuration
+
+MTA supports Red Hat Single Sign-On (RHSSO) configuration, so that users' roles can be added using `keycloak`to control Konveyor and MTA authorization mechanisms. See link:https://access.redhat.com/documentation/en-us/migration_toolkit_for_applications/6.0/html/user_interface_guide/mta-6-installing-web-console-on-openshift_user-interface-guide#red_hat_single_sign_on[Red Hat Single Sign-On]. 
+
+
+.New Rules Syntax
+
+New rules syntax in YAML supports metadata, `message` and `tag` actions, rule conditions, provider conditions for `java` and `go` providers, and other file tagging and characteristics.
+
+
+.Filename of downloaded report file matches application name
+
+Downloaded files are now named according to the migrated application.
+
+
+.Tag Category drop down menu shows only categories relevant to the source
+
```

Comment:
IBM Style Guide, "Usage and highlighting for interface elements" -- Avoid using drop-down. So here, you'd want to refer to the Tag category (or is it categories?) menu (or is it list?). Please double-check the UI for the item's exact name, including capitalization, and UI element (menu vs. list).